specifyweb/frontend/js_src/lib/localization/backEnd.ts

```diff
@@ -10,6 +10,11 @@ import { createDictionary } from './utils';
 // Refer to "Guidelines for Programmers" in ./README.md before editing this file
 
 export const backEndText = createDictionary({
+
+  ///////////////////////
```

Comment:
note, these comments (however pretty they look) would be removed by weblate when it updates the local strings from the remote.

If you need to add a comment for an individual string, add a `comment` key. Note, the value of the `comment` key is visible to translators.

Example:

```js
{
  ...
  failedParsingBoolean: {
    comment: `
     Some comment.
     Visible to translators.
            not white     space        sensitive

     Blank line is treated as a multi-line.
    `,
    'en-us': '...',
    ...
  },
  ...
}